macroslib/src/python/python-include.rs

```diff
@@ -0,0 +1,29 @@
+
+#[macro_use] extern crate cpython;
+use cpython::{ObjectProtocol, PyObject, PythonObjectWithTypeObject};
```

Comment:
ObjectProtocol looks too common name for code that mixed with
potentially any code provided by user. May be use full name everywhere in generated code,
like ::cpython::ObjectProtocol

Reply:
It is extension trait and must be imported with "use". I renamed it to "CPythonObjectProtocol" to avoid potential name collisions.